packages/kit/src/runtime/server/endpoint.js

```diff
@@ -29,10 +29,7 @@ export default async function render_route(request, route) {
 
 			headers = lowercase_keys(headers);
 
-			if (
-				(typeof body === 'object' && !('content-type' in headers)) ||
-				headers['content-type'] === 'application/json'
-			) {
+			if (typeof body === 'object' && !('content-type' in headers)) {
```

Comment:
should this keep the old condition, but with the parens in a different place? I.e.
```
if (typeof body === 'object' && (!('content-type' in headers) || headers['content-type'] === 'application/json'))
```

Reply:
done.